view/next-project/src/pages/budgets/index.tsx

```diff
@@ -168,50 +189,51 @@ export default function BudgetList(props: Props) {
                       </tr>
                     </thead>
                     <tbody>
-                      {filteredBudgets.map((budgetView, index) => (
-                        <tr
-                          key={budgetView.budget.id}
-                          className={clsx(
-                            index !== budgets.length - 1 && 'border-b',
-                            'py-3 text-black-600',
-                          )}
-                        >
-                          <td className='py-3 text-center'>{budgetView.source.name}</td>
-                          <td className='py-3 text-center'>{budgetView.year.year}</td>
-                          <td className='py-3 text-center'>{budgetView.budget.price}</td>
-                          <td className='py-3 text-center'>
-                            {formatDate(
-                              budgetView.budget.createdAt ? budgetView.budget.createdAt : '',
-                            )}
-                          </td>
-                          <td className='py-3 text-center'>
-                            {formatDate(
-                              budgetView.budget.updatedAt ? budgetView.budget.updatedAt : '',
+                      {budgetViews &&
+                        budgetViews.map((budgetView, index) => (
+                          <tr
+                            key={budgetView.budget.id}
+                            className={clsx(
+                              index !== budgets.length - 1 && 'border-b',
```

Comment:
```suggestion
                              budgets && index !== budgets.length - 1 && 'border-b',
```
![スクリーンショット 2024-02-22 153031](https://github.com/NUTFes/FinanSu/assets/131145590/a5d8607d-33c7-4380-a7ec-94c4b12931af)


budgetsがnullなときにエラーが出ていました！

Reply:
修正しました
[[fix]バグ修正](https://github.com/NUTFes/FinanSu/pull/681/commits/77feb1c9d47f11f29c03f4c567985195627126c7)